xml/admin_upgrade.xml

```diff
@@ -256,4 +256,213 @@ value: value)}'</command></screen>
    </procedure>
   </sect2>
  </sect1>
+ 
+ <sect1 xml:id="sec.deploy.upgrade.caasp2">
+  <title>Upgrading from &productname; 2</title>
+  <note>
+   <para>
+    If you wish to upgrade to &productname; v3 from v1 you must first perform
+    all the steps described in <xref linkend="sec.deploy.upgrade.caasp1"/>.
+   </para>
+  </note>
+  
+  <sect2 xml:id="sec.deploy.upgrade.caasp2.prereq">
+   <title>Ensure All &productname; v2 Updates Are Installed</title>
+   <para>
+    Before you start the upgrade procedure to &productname; v3, you must ensure
+    that all your nodes are running on the latest v2 updates. You can check the
+    <filename>SUSEConnect</filename> package version to see if you are up to date.
+    To do so you will run a <command>salt</command> command to display the
+    package version installed on each node.
+   </para>
+   
+   <screen>
+&prompt.user;<command>docker exec -i  $(docker ps | grep salt-master | awk '{print $1}') \
+salt --batch 10 -P "roles:(admin|kube-(master|minion))" \
+cmd.run "rpm -q SUSEConnect"</command>
+
+Executing run on ['12cda3c374144d74804298bdee4d686c',
+                  '9b6d8d28393045c0914c959d0a5c0e33',
+                  '73b92dd7816147058c3d0fbb67fb18f9',
+                  'admin']
+admin:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+73b92dd7816147058c3d0fbb67fb18f9:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+9b6d8d28393045c0914c959d0a5c0e33:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+12cda3c374144d74804298bdee4d686c:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+   </screen>
+   
+   <para>
+    If the package version is <literal>0.3.11-3.15.1</literal> (or higher) you
+    have the latest updates from the v2 channel installed.
```

Comment:
i guess that's a good way to know if you're up-to-date, but only if there's no update to SUSEConnect in the meantime... is there maybe a way to run a suseconnect query that tells if we're on the latest v2 head?

Reply:
@MaximilianMeister Can you provide me with the command needed to do this and some sample output? If no, who can provide this? This was @kiall suggestion to do a simple check if you have the latest packages.

Reply:
maybe `SUSEConnect --status` ? i have to try it on some system